app/views/admin/evaluations/_restitution_globale.arb

```diff
@@ -96,7 +98,7 @@ div class: 'evaluation__restitution-globale' do
            }
   end
 
-  if cafe_de_la_place
+  if restitution_globale.synthese_positionnement.present?
```

Comment:
Je comprends le sens d'avoir ajouté une valeur dans le cas indéterminé, mais cette condition est in peu déroutante. Je trouvais la condition précédente plus explicite.
Je me dis que l'idéal serait peut-être d'écrire : 
`if campagne_positionnement?` un peu comme on a écrit plus haut `if campagne_avec_competences_transversales?`

Reply:
Le problème avec `if campagne_avec_competences_transversales?` c'est qu'aujourd'hui on affiche le bloc même si on a pas de restitution. Est-ce que vaudrait pas mieux du coup rester sur une condition sur la restitution plutôt que sur la campagne ?
Je suis d'accord que la condition cafe_de_la_place était mieux.